src/evict/evict_page.c

```diff
@@ -663,34 +663,62 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
     /* Make sure that both conditions above are not true at the same time. */
     WT_ASSERT(session, !use_snapshot_for_app_thread || !is_eviction_thread);
 
-    if (!conn->txn_global.checkpoint_running && !WT_IS_HS(btree->dhandle) &&
-      (use_snapshot_for_app_thread || is_eviction_thread)) {
+    /* History store data are always evictable. */
+    if (WT_IS_HS(btree->dhandle)) {
+        /* TODO: remove this when we have removed history store score and __rec_update_stable. No
```

Comment:
Comment formatting is wrong here also I think a TODO should be FIX-ME-TICKET_NUMBER format do we have a ticket to record here?

Reply:
There should be one I think.